### DISCUSSION TITLE
ennexPortal

### DISCUSSION THREAD

**Discussion by trsen** [camueller/SmartApplianceEnabler#604] [Installation und Betrieb]
Ist die Funktion des SEA auch uneingeschränkt mit dem ennexPortal gegeben ? Neue Anlagen von SMA können nur noch mit dem neuem Portal registriert werden.

**Comment by camueller** [comment]
Der SAE kommuniziert nur mit dem Sunny Home Manager insofern hat der Wechsel beim Portal keine Auswirkungen.
